### PR TITLE
Introduce templates and codeowners for github

### PR DESCRIPTION
**Pull request by benoitf**
Following discussion on #openshiftio-che channel, here is a pull request

- Introducing template for issues and pull requests
https://github.com/blog/2111-issue-and-pull-request-templates

- Introducing code owners file
https://github.com/blog/2392-introducing-code-owners

BTW who is admin of this repository to check the boxes to ask for a reviewer for merging 
https://github.com/blog/2392-introducing-code-owners#an-extra-layer-of-code-security

note: let's commit to this branch to do your changes/suggestions

Change-Id: I52c742720fc74c419357bd20cb44fdbc937da53a
Signed-off-by: Florent BENOIT <fbenoit@redhat.com>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!--- 
+Replace this comment with a description of the problem.
+
+Please help us by providing all of the details listed below. A complete and thoughtful issue request with research on possible root causes and diagnostics goes a very long way in helping us rapidly get your issues resolved. 
+--> 
+
+**Reproduction Steps:**
+
+**OS and version:**    
+
```
Comment (l0rd) [inline .github/ISSUE_TEMPLATE.md]: I think that some important info that we need here are:
- rhche version
- openshift and k8s versions

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+### What does this PR do?
+
+
+### What issues does this PR fix or reference?
+
+
+#### Changelog
+<!-- Write a short (one line) entry to summarize the changes of this PR -->
```
Comment (l0rd) [inline .github/PULL_REQUEST_TEMPLATE.md]: Another question I would add:
- How have you tested this PR?

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Global Owners
+
```
Comment (sunix) [inline .github/CODEOWNERS]: So who are the global owners ?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (amisevsk) [inline .github/PULL_REQUEST_TEMPLATE.md]: +1 on steps for testing changes introduced

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (amisevsk) [inline .github/ISSUE_TEMPLATE.md]: Maybe more explicit language is helpful:
- are you running on minishift or something else
- if minishift include output of `minishift version`
- include output of `oc version`
- rhche version / does it reproduce on latest official image

**Comment by benoitf** [comment]
thx for the feedback, we may discuss it at the F2F ?

**Comment by benoitf** [comment]
FYI I have updated the templates

**Comment by benoitf** [comment]
@l0rd is it ok to introduce it and then we can iterate with feedback ?

**Pull request merged**